docs/visualization/plotting/plotting-nd-data.ipynb

```diff
@@ -24,9 +24,22 @@
    "metadata": {},
    "outputs": [],
    "source": [
+    "%matplotlib widget\n",
     "import numpy as np\n",
-    "import scipp as sc\n",
-    "%matplotlib widget"
+    "import scipp as sc"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "nbsphinx": "hidden",
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "sc.plot = sc.plotting.plot\n",
+    "sc.DataArray.plot = sc.plotting.plot"
```

Comment:
Use `unpatch_scipp`?

Reply:
Then notebook would depend on plopp, I wanted to avoid that.